### ISSUE TITLE
Add modalizer `isTrapped` option to `useModalAttributes` in react-tabster and react-popover

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#23599]
Pick up the changes in microsoft/tabster#122 to add the traditional full focus trap to react-tabster.

react-popover should be able to be configured to this focus trap variant
